### PR TITLE
feat(Stack): Support `between`, `evenly` and `around` space

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

File: src/__stories__/stack-story.tsx [hunk 1]
```diff
@@ -1,16 +1,29 @@
 import * as React from 'react';
```
Comment (atabel) [inline src/__stories__/stack-story.tsx:1]: please, review this story in dark mode, the text isn't readable

File: src/__stories__/stack-story.tsx [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
-import {Stack, Text2, useTheme} from '..';
-import {StorySection, useSelect} from './helpers';
-import {Placeholder} from '../placeholder';
+import {Stack, Text2} from '..';
 
 export default {
     title: 'Components/Layouts/Stack',
+    component: Stack,
+    argTypes: {
```
Comment (atabel) [inline src/__stories__/stack-story.tsx:7]: If we are using args here, we could do the same with the Inline story

**Review by atabel — APPROVED** [review]

**Review by yceballost — APPROVED** [review]
nice!

**Pull request merged**